q2_types/ordination/tests/test_transformer.py

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import unittest
+
+import skbio
+
+from q2_types.ordination import OrdinationFormat
+from qiime.plugin.testing import TestPluginBase
+
+
+class TestTransformers(TestPluginBase):
+    package = 'q2_types.ordination.tests'
+
+    def test_skbio_ordination_results_to_ordination_format(self):
+        filepath = self.get_data_path('pcoa-results.txt')
```

Comment:
Can you use the 3 sample distance matrices in `q2_types.distance_matrix.tests.data` to create 3 matching `pcoa-results.txt` files? We would use the `NxN` variant in the `test_format.py` tests, and then here in `test_transformer.py` we would loop over them, similar to the [distance matrix tests](https://github.com/qiime2/q2-types/blob/master/q2_types/distance_matrix/tests/test_transformer.py#L23-L25).